src/objects/domain_obj.f90

```diff
@@ -10,7 +10,7 @@
 submodule(domain_interface) domain_implementation
     use assertions_mod,       only : assert, assertions
     use mod_atm_utilities,    only : exner_function, update_pressure
-    use icar_constants,       only : kVARS, kLC_LAND
+    use icar_constants,       !only : kVARS, kLC_LAND
```

Comment:
I'd like to keep the `only : ` as much as possible.  Can you just add the additional constants you are using (`kLC_WATER` and `kWATER_LAKE`?)